mne/channels/channels.py

```diff
@@ -950,6 +950,111 @@ def interpolate_bads(
 
         return self
 
+    def interpolate_to(self, montage, origin="auto", method="spline", reg=0.0):
+        """Interpolate EEG data onto a new montage.
+
+        Parameters
+        ----------
+        montage : DigMontage
+            The target montage containing channel positions to interpolate onto.
+        origin : array-like, shape (3,) | str
+            Origin of the sphere in the head coordinate frame and in meters.
+            Can be ``'auto'`` (default), which means a head-digitization-based
+            origin fit.
+        method : str
+            Method to use for EEG channels.
+            Supported methods are 'spline' (default) and 'MNE'.
+
+            .. warning::
+                Be careful, only EEG channels are interpolated. Other channel types are
+                not interpolated.
```

Comment:
It looks like this warning applies to the entire function, not the `method` parameter specifically. Perhaps move it to the main text, so above the `Parameters` line.